src/extensions/default/QuickView/main.js

```diff
@@ -400,15 +400,18 @@ define(function (require, exports, module) {
                 }
             } else if (pos.ch <= match.index + match[0].length) {
                 // build the css for previewing the gradient from the regex result
-                var previewCSS = gradientMatch.prefix + (gradientMatch.colorValue || match[0]);
+                var previewCSS = gradientMatch.prefix + (gradientMatch.colorValue || match[0]); 
+                
+                if ((/^0x/).test(previewCSS)) {
```

Comment:
can this be a function like:
```
function ensureHexFormat(str) {
  return (/^0x/).test(str) ? str.replace("0x","#") : str;
}
```
adding constructs like these without explanation makes the code less readable to me